packages/openapi/lib/decorators.tsp

```diff
@@ -30,7 +30,7 @@ extern dec operationId(target: Operation, operationId: valueof string);
  * op read(): string;
  * ```
  */
-extern dec extension(target: unknown, key: valueof string, value: unknown);
+extern dec extension(target: unknown, key: valueof string, value: unknown | (valueof unknown));
```

Comment:
should we remove the `unknown` option so it warns when trying to use model and allows use to have the schema conversion later?

Reply:
I kept it in so I could display a more targeted warning for anyone passing in types instead of values, but if that feels unnecessary I can definitely switch this to just `valueof unknown` for this PR!

Reply:
Went with your suggestion - having access to the codefix is really nice.